privacy-policy.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: default
+permalink: /privacy-policy
+is_markdown: true
+---
+
+_Last updated: 18 March 2019_
+
+## Introduction
+
+We take the protection of our users (“User/you/your”) personal data very seriously and strictly comply with applicable data protection laws and regulations. In our privacy policy below (“Privacy Policy”) we provide you with an overview of what data we collect for what purpose and how we ensure the protection of the data.
+
+The controller is Gaiwan GmbH, c/o Arne Brasseur, Kinzigstr. 26, 10247 Berlin/Germany (“we/us/our”).
+
+In general Gaiwan GmbH aims to only store, process, or share the minimum information necessary. The less we know about you, the better. We do need to know certain things in order to provide services, communicate with you, or meet legal requirements. In this document you can find a complete overview of what we collect and store.
+
+## Information gathering and usage
+
+This privacy policy pertains to all online assets and services that are part of the official Heart of Clojure conference organization. These are:
+
+- https://heartofclojure.eu ("the website")
+- https://cfp.heartofclojure.eu ("the CFP app")
+- https://activities.heartofclojure.eu ("the activities app")
+
+We also make use of third party services to collect, store, and process information. These are:
+
+- [MailChimp](https://mailchimp.com)
+- [Tito](https://ti.to)
```

Comment:
Google is missing here. We use it for some stuff. I think so far not for user-submitted data but that may change, especially with additional forms to collect information from speakers.

Reply:
Given that we're not using this right now I also can't describe how we're using it. I'm also not sure at this point what we would do with it re. personal data, so I'm gonna leave this out. We can update the privacy policy if necessary.